modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/command/response/ChecksumInfo.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.metastorage.command.response;
+
+import java.io.Serializable;
+import org.apache.ignite.internal.tostring.S;
+
+/**
+ * Information about checksum for a revision of the Metastorage.
+ */
+public class ChecksumInfo implements Serializable {
```

Comment:
I think this class and corresponding commands should be called `RevisionInfo`, because this is information about a requested revision, not a checksum

Reply:
The focus here is on the checksum. Also, it contains min/max revisions as well, which are not about the target revision.

`ChecksumInfo` seems better as it defines the context.